boms/user/tools/pom.xml

```diff
@@ -68,6 +77,14 @@
                             <bomName>${full.dist.product.release.name} BOMs: Tools</bomName>
                             <bomDescription>Dependency Management with Tools useful for ${full.dist.product.release.name}</bomDescription>
                             <inheritExclusions>NONE</inheritExclusions>
+                            <channels>
+                                <channel>
+                                    <manifest>
+                                        <groupId>org.wildfly.channels</groupId>
+                                        <artifactId>wildfly-ee</artifactId>
+                                    </manifest>
+                                </channel>
+                            </channels>
```

Comment:
This seems off. I don't think our channels should be controlling the tools stuff. Is it for things like the CLI, which is a tool that also gets provisioned?

Reply:
Yes, and there are other artifacts like httpclient too, all come from EE (we moved all from ee-with-tools) thus the usage of ee channel.

Reply:
Ok re the channel.

I realize now that standard WildFly controls these versions, even in the wildfly-ee|expansion-preview-with-tools boms.

Not ideal, but ok.